backend/src/components/application.js

```diff
@@ -178,7 +178,7 @@ async function upsertParentFees(req, res) {
         ccof_frequency: feeGroup.feeFrequency,
         'ccof_ChildcareCategory@odata.bind': childCareCategory,
         'ccof_ProgramYear@odata.bind': programYear,
-        'ccof_ApplicationCCFRI@odata.bind': `/ccof_applicationccfris(${feeGroup.ccfriApplicationGuid})`,
+        //'ccof_ApplicationCCFRI@odata.bind': `/ccof_applicationccfris(${feeGroup.ccfriApplicationGuid})`,
```

Comment:
is it for the PROD issue fix? I thought you want to keep it?

Reply:
yes!! good catch - we def want to keep it.